pkg/model/types.go

```diff
@@ -33,12 +33,20 @@ type Service struct {
 	Endpoints []*Endpoint
 }
 
+const (
+	HeadlessType  ServiceType = "Headless"
+	ClusterIPType ServiceType = "ClusterSetIP"
```

Comment:
After looking at it again, I think it may be less confusing if we define it as exporting a `ServiceType`, rather than the `ServiceImportType`. Ultimately, the attribute is meant to define the type of service it is exporting, not the type of `ServiceImport` it aims to create. 
I think then the value should be `ClusterIP`, rather than `ClusterSetIP`.

Reply:
I have to  disagree. I think we are defining the service type from the mcs spec so it should follow the same convention.

Reply:
I see what you're saying; will make the changes accordingly.